rs/backend/src/tvl/tests.rs

```diff
@@ -0,0 +1,57 @@
+use crate::tvl::{self, governance};
+
+fn get_total_locked_icp_e8s() -> u64 {
+    tvl::STATE.with(|s| s.tvl_state.borrow().total_locked_icp_e8s)
+}
+
+fn set_total_locked_icp_e8s(new_value: u64) {
+    tvl::STATE.with(|s| s.tvl_state.borrow_mut().total_locked_icp_e8s = new_value);
+}
+
+#[tokio::test]
+async fn update_locked_icp_e8s() {
+    let initial_locked_icp_e8s = 50_000_000_000;
+    let later_locked_icp_e8s = 90_000_000_000;
+
+    set_total_locked_icp_e8s(initial_locked_icp_e8s);
+    assert_eq!(get_total_locked_icp_e8s(), initial_locked_icp_e8s);
+
+    governance::testing::add_metrics_response_with_total_locked_e8s(later_locked_icp_e8s);
+
+    tvl::update_locked_icp_e8s().await;
+
+    assert_eq!(get_total_locked_icp_e8s(), later_locked_icp_e8s);
```

Comment:
NCR

```suggestion
    // Step 1: Prepare the world.

    let initial_locked_icp_e8s = 50_000_000_000;
    let later_locked_icp_e8s = 90_000_000_000;

    set_total_locked_icp_e8s(initial_locked_icp_e8s);
    assert_eq!(get_total_locked_icp_e8s(), initial_locked_icp_e8s);

    governance::testing::add_metrics_response_with_total_locked_e8s(later_locked_icp_e8s);

    // Step 2: Call the code under test.
    tvl::update_locked_icp_e8s().await;

    // Step 3: Inspect result(s).
    assert_eq!(get_total_locked_icp_e8s(), later_locked_icp_e8s);
```

I think it's helpful to be super clear which phase of a test we are in.

Reply:
Done